website/addons/wiki/model.py

```diff
@@ -43,6 +81,23 @@ def after_register(self, node, registration, user, save=True):
             clone.save()
         return clone, None
 
+    def after_set_privacy(self, node, permissions):
+        """
+
+        :param Node node:
+        :param str permissions:
+        :return str: Alert message
+
+        """
+        if permissions == 'private':
+            if self.is_publicly_editable:
+                self.set_editing(False)
```

Comment:
Even though `log=False` by default, it would be informative to pass `log=False` explicitly here. This will also ensure that we don't add an extra log if we ever change the default.